test/benchmarks/benchmark_compare256.cc

```
@@ -3,19 +3,16 @@
  * For conditions of distribution and use, see copyright notice in zlib.h
  */
 
-#include <stdio.h>
-
 #include <benchmark/benchmark.h>
```

Cppcheck (2.19.0)

[information] Include file

(missingIncludeSystem)